packages/libs/error-stack/Cargo.toml

```diff
@@ -22,6 +22,7 @@ anyhow = { version = "1.0.65", default-features = false, optional = true }
 eyre = { version = "0.6", default-features = false, optional = true }
 owo-colors = { version = "3", default-features = false, optional = true, features = ['supports-colors'] }
 serde = { version = "1", default-features = false, optional = true }
+spin = { version = "0.9.4", default-features = false, optional = true, features = ['rwlock', 'once'] }
```

Comment:
Does `0.9` also work for our use cases?

Reply:
requirement relaxed in [122a119](https://github.com/hashintel/hash/pull/1556/commits/122a119a12021b17f17782443d0422b47826f166)

Reply:
Great, thanks!